tokens/Cargo.toml

```diff
@@ -1,43 +1,40 @@
 [package]
-name = "orml-tokens"
+authors = ["Laminar Developers <hello@laminar.one>"]
```

Comment:
why changing this?

Reply:
it's automatically done by editor (formatOnSave)